DynamicLights/SpikingNet.cpp

```diff
@@ -713,6 +717,19 @@ bool SpikingNet::getFlagDecay() const {
     return this->flagDecay;
 }
 
+void SpikingNet::writeNetworkType(SpikingNet::NetworkType networkType) {
+    if (this->networkType == networkType)
+        return;
+
+    // reset network, since these parameters only take effect when the network is created anew
+    // note from Julien: shouldn't the network be reinitialized AFTER changing the values?
+    reset();
+    initialize();
+
+    this->networkType = networkType;
+    emit networkTypeChanged(networkType);
+}
+
```

Comment:
Network types will get deprecated when we implement the drawing interface, since the only network type that will work with this is the grid network. We don't need to have them be a part of the UI.